SmartDeviceLinkTests/RPCSpecs/StructSpecs/SDLStabilityControlsStatusSpec.m

```diff
@@ -0,0 +1,68 @@
+//
+//  SDLStabilityControlsStatusSpec.m
+//  SmartDeviceLink
+
+
+#import <Foundation/Foundation.h>
+
+#import <Quick/Quick.h>
+#import <Nimble/Nimble.h>
+
+#import "SDLStabilityControlsStatus.h"
+#import "SDLRPCParameterNames.h"
+#import "SDLVehicleDataStatus.h"
+
+
+QuickSpecBegin(SDLStabilityControlsStatusSpec)
+
+describe(@"Getter/Setter Tests", ^ {
+    it(@"Should set and get correctly (On)", ^ {
+        SDLStabilityControlsStatus* testStruct = [SDLStabilityControlsStatus new];
+        testStruct.escSystem = SDLVehicleDataStatusOn;
+        testStruct.trailerSwayControl = SDLVehicleDataStatusOn;
+        expect(testStruct.escSystem).to(equal(SDLVehicleDataStatusOn));
+        expect(testStruct.trailerSwayControl).to(equal(SDLVehicleDataStatusOn));
+    });
+
+    it(@"Should set and get correctly (Off)", ^ {
+        SDLStabilityControlsStatus* testStruct = [SDLStabilityControlsStatus new];
+        testStruct.escSystem = SDLVehicleDataStatusOff;
+        testStruct.trailerSwayControl = SDLVehicleDataStatusOff;
+        expect(testStruct.escSystem).to(equal(SDLVehicleDataStatusOff));
+        expect(testStruct.trailerSwayControl).to(equal(SDLVehicleDataStatusOff));
+    });
+    
+        it(@"Should get correctly when initialized with a dict (On)", ^ {
+            NSDictionary* dict = @{@"escSystem":SDLVehicleDataStatusOn,
+                                   @"trailerSwayControl":SDLVehicleDataStatusOn};
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
+            SDLStabilityControlsStatus* testStruct = [[SDLStabilityControlsStatus alloc] initWithDictionary:dict];
+#pragma clang diagnostic pop
+
+            expect(testStruct.escSystem).to(equal(SDLVehicleDataStatusOn));
+            expect(testStruct.trailerSwayControl).to(equal(SDLVehicleDataStatusOn));
+        });
+
+    it(@"Should get correctly when initialized with a dict (Off)", ^ {
+        NSDictionary* dict = @{@"escSystem":SDLVehicleDataStatusOff,
+                               @"trailerSwayControl":SDLVehicleDataStatusOff};
```

Comment:
`SDLParameterNameEscSystem ` `SDLParameterNameTrailerSwayControl `

Reply:
If I replace the strings with constants declared in another file how can I be sure the constants are correct?
In such case the test will test nothing will it not?